### DISCUSSION TITLE
Outdoor temperature sensor KO

### DISCUSSION THREAD

**Discussion by emilic14** [pcmihnea/HA_WMR500#12] [Q&A] [answered]
Hello!

So... I've got an issue ^^
my external temperature is now reading 0 ALL the time...

anyone has any experience fixing this ? 

for the story:
My device was and has been working great for about 1 year.
Then, I left my device outdoor lying on the floor during humid times...
Sudently the temperature sensor started jumping from 0 to the real value and back (it was quite random).
So I opened my device and "cleaned" the sensor. I got back my measurements YEII!!!
But after a few days, same issue again, and well now it just plain stoped and is now 0 all the time...

I've disassembled the whole device ^^ 

![WhatsApp Image 2023-07-11 at 10 26 22 PM](https://github.com/pcmihnea/HA_WMR500/assets/4341271/a7d6e2c8-deec-463e-8f4b-f3166c5ce369)

and cleaned AGAIN but still nothing.
Voltage getting to the sensor seems ok (2.4v).
connections seem ok.
Resistors or capacitors I am not sure ^^

![WhatsApp Image 2023-07-11 at 10 26 21 PM](https://github.com/pcmihnea/HA_WMR500/assets/4341271/3531b927-5b4b-42db-b131-65bef535a5aa)

And I forgot, for the moment I am avoiding putting it outdoors as I realized that all the electronic devices are covered with a small "water protection" spray to avoid oxidation etc... So until I fix the issue and spray it again, Its not going out

**Comment by pcmihnea** [comment]
Hi,

Unfortunately the most fragile sensor is also the least protected one - the chip's humidity measurement window doesn't have any waterproofing.
Is the null value only from the temp/humidity, or the whole external device?
You could exchange the chip itself with the one used in the main base, AFAIK they're the same part number.

**Reply by emilic14** [reply]
Yeah no waterproofing per se... so recommendation : NEVER let your station laying down on its side in a humid place... 

as far as I can see the only nul (0.0) value is of the temperature. I am registering wind and rain.

Do you know which model would the temperature chip be ? as in, I think rather than getting the one from the internal device (which is working) I could get a replacement one and just change it ^^... That should work I guess...

But I think that I first need to check if the resistors are not faulty or the capacitors.

**Reply by pcmihnea** [reply]
Is the humidity still being measured?
I Can't help much with the product number, perhaps you could sgoogle search the markings on the chip, but even then it would be a hassle to replace just the chip - the PCB seems custom made for this product.

**Reply by emilic14** [reply]
It is not.

I'll do some test and update my findings :)

**Comment by emilic14** [comment]
Well small update...
I am doing a small sketch of the circuit... and removed the resistors and capacitors...
While measuring them I can see that the 2 capacitors seem to be 
~1200nf
and the 2 resistors are
~1k ohm

Luckily for me, it seems that 1 resistor is not working...
So I am going to replace it and see if the system gets back to life.

**Reply by pcmihnea** [reply]
Hi, the capacitor is most likely as a power supply filter, so it's value isn't that critical. The two resistors may be pull-ups for the i2c lines, so should be kept the same.

**Reply by emilic14** [reply, marked as answer]
Success!!! 
It was the resistor. I put one one the fly (while I get an SMD one) to test and it worked!!

![WhatsApp Image 2023-07-22 at 2 36 50 PM](https://github.com/pcmihnea/HA_WMR500/assets/4341271/f62fb5d6-eecb-4137-82f7-61ff0ecd197d)
![WhatsApp Image 2023-07-22 at 2 36 50 PM (1)](https://github.com/pcmihnea/HA_WMR500/assets/4341271/8048fada-0622-4632-86d8-5cceb98e1571)

Next step will be to put an SMD one (so that I can put the sensor back in place)
and to apply some insulating varnish so that it can live outside :)

**Reply by pcmihnea** [reply]
Nice, that's honestly the best scenario, good thing the chip is still alive.
